src/convert/transformers/decomposedMetadataTransformer.ts

```diff
@@ -74,6 +75,30 @@ export class DecomposedMetadataTransformer extends BaseMetadataTransformer {
             const source = new JsToXml({
               [childType.name]: Object.assign({ [XML_NS_KEY]: XML_NS_URL }, value),
             });
+
+            /*
+             composedMetadata is a representation of the parent's xml
+             if there is no CustomObjectTranslation in the org, the composedMetadata will be 2 entries
+             the xml declaration, and a fields attribute, which points to the child CustomObjectFieldTranslation
+             because CustomObjectFieldTranslation is the only metadata type with 'requiresParent' = true we can
+             calculate if a CustomObjectTranslation was retrieved from the org (composedMetadata.length > 2), or,
+             if we'll have to write an empty CustomObjectTranslation file (composedMetadata.length <=2).
+             CustomObjectFieldTranslations are only addressable through their parent, and require a
+             CustomObjectTranslation file to be present
+             */
+            if (childType.requiresParent && composedMetadata.length <= 2) {
```

Comment:
this could use the `isAddressable` field, because COFT is the only metadata type to set either of those. It just seemed like the naming and intention could become confusing if I used `isAddressable` 🤷‍♂️

Reply:
I'm gonna defer to Steve's opinion on this one.

My question is whether it's valid to have `isAddressable:false` without `requiresParent:true`, or vice versa?  

Or would you say that what we really meant was `unaddressableWithoutParent` ?

Reply:
I know there are more entities that are unaddressable, and that field is useful and makes sense so that one should remain.  I agree with Shane that `unaddressableWithoutParent` is better.  COT and COFT are unfortunate anomalies that hopefully will never happen again.  `unaddressableWithoutParent` is the nicest description of this disastrous pair.